python/cookiecutter-django.md

```diff
@@ -147,21 +150,28 @@ READ_DOT_ENV_FILE = env.bool("DJANGO_READ_DOT_ENV_FILE", default=True)
 #  .env.example
 
 DJANGO_DEBUG=on
-DATABASE_URL=postgres://<user>:<password>@localhost:5432/cookiecutter_starter-dev
+DATABASE_URL=postgres://<user>:<password>@localhost:5432/<db-name-dev>
 ```
 
+For exampple, let's say you have:
+- A created db named `cookiecutter_starter-db-dev`, and a user `postgres` with password `postgres`. 
+- Then the value for `DATABASE_URL` would be: `DATABASE_URL=postgres://postgres:postgres@localhost:5432/cookiecutter_starter-db-dev`.
+
+### Important Note
+For the next commands, we will assume that you are in the root folder, and you have already entered to the sell with `$ pipenv shell`.
+
 ## Migrate
-`$ pipenv run python manage.py migrate`
+`$ python manage.py migrate`
 
 ## Run test
-`$ pipenv run pytest`
+`$ pytest`
 
 ## Run test and get coverage percentage
-`$ pipenv run coverage run -m pytest && pipenv run coverage report -m`
+`$ coverage run -m pytest && pipenv run coverage report -m`
 
 NOTE:
-- The command must run without exceptions because it mandatory for future configurations.
-- Don't matter if you write tests following the [unittest](https://docs.python.org/3/library/unittest.html) or [pytest](https://docs.pytest.org/) approach since `$ pipenv run pytest` command will be able to run both of them.
+- The command must run without exceptions because it's mandatory for future configurations.
+- Don't matter if you write tests following the [unittest](https://docs.python.org/3/library/unittest.html) or [pytest](https://docs.pytest.org/) approach since `$ pytest` command will be able to run both of them.
```

Comment:
What happened when you tried to use `pipenv` command?

Reply:
What do you mean?